### ISSUE TITLE
[Adaptive Cards> InputFormwithRTL]: Lang attribute is not defined for the 'Name, Email, Phone and Comments' edit fields. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8974]
### Target Platforms

UWP

### SDK Version

1.0.1.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Universal
OS Build: 24H2 (OS Build 26252.5000)
App Version: 1.0.1.0
Screen Reader: Narrator

**Repro Steps:**
1. Open the Adaptive Cards Universal application.
2. Adaptive Card UWP Visualizer screen starts appearing.
3. Navigate to InputFormWith RTL.Json button by using Tab key and activate it.
4. InputFormwithRTL adaptive card will appear on right sight of the pane.
5. Navigate to InputFormwithRTL card by using Tab key.
6. Press tab key and navigate to the Arabic button and press enter key to activate it.
7. Navigate the control and observe the issue.

**Actual Result:**
Lang attribute is not defined for the 'Name, Email, Phone and Comments' edit fields. When the screen reader focuses on 'Name'  edit field It is announcing as Edit required.

**Expected Result:**
Lang attribute should be defined for the 'Name, Email, Phone and Comments' edit fields which is presented in different language from the rest of the page. When focus lands on the edit fields, it should announce the control information. It should announce as 'Name edit field required.'

**Note:**
1. Same issue is repro with NVDA screen reader.
2. Same issue is repro for the Arabic button as well.

**User Experience:**
If the language of part is not defined for the content defined in the list, then the user who depends upon Screen Reader for navigation might get affected and hence fail to proper information of the control present on page.

**WCAG Reference Link:**
https://www.w3.org/WAI/WCAG21/Understanding/language-of-parts.html

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

![Lang attribute is not defined for the 'Name, Email, Phone and Comments' edit fields](https://github.com/user-attachments/assets/cddc22cd-9323-4dbc-82e6-90a018ca5c3b)

https://github.com/user-attachments/assets/df0b3257-e11e-4c69-b8b3-fb01a90dbdf4

![Web - Lang attribute is not defined for the 'Name, Email, Phone and Comments' edit fields](https://github.com/user-attachments/assets/f65dcc7d-f441-4106-b686-981c5a33a0ac)

### Card JSON

```JSON
NA
```
